src/ports/postgres/modules/svm/svm.py_in

```diff
@@ -144,30 +145,37 @@ def _verify_get_params_dict(params_dict):
 
 
 def _build_output_tables(n_iters_run, model_table, args, transformer, **kwargs):
-    if transformer is None:
+    # transformer should always be a valid object
+    original_table = transformer.original_table
+    if not original_table:
```

Comment:
Do you mean `original_table` is always a non-empty dictionary? If that is a restriction, then we'll have to add a note in the `kernel_approximation` classes ensuring that future kernels never set `original_table` to empty.